cmd/expiration-mailer/main.go

```diff
@@ -239,25 +240,51 @@ func (m *mailer) findExpiringCertificates() error {
 
 		m.log.Info(fmt.Sprintf("expiration-mailer: Searching for certificates that expire between %s and %s and had last nag >%s before expiry",
 			left.UTC(), right.UTC(), expiresIn))
+
+		// If CertStatusOptimizationsMigrated is enabled then we can do this query
+		// in a slightly more efficient way by looking at `cs.NotAfter` instead of
+		// `cert.expires` in the WHERE clause.
 		var certs []core.Certificate
-		_, err := m.dbMap.Select(
-			&certs,
-			`SELECT cert.* FROM certificates AS cert
-			 JOIN certificateStatus AS cs
-			 ON cs.serial = cert.serial
-			 AND cert.expires > :cutoffA
-			 AND cert.expires <= :cutoffB
-			 AND cs.status != "revoked"
-			 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
-			 ORDER BY cert.expires ASC
-			 LIMIT :limit`,
-			map[string]interface{}{
-				"cutoffA":   left,
-				"cutoffB":   right,
-				"nagCutoff": expiresIn.Seconds(),
-				"limit":     m.limit,
-			},
-		)
+		var err error
+		if features.Enabled(features.CertStatusOptimizationsMigrated) {
+			_, err = m.dbMap.Select(
```

Comment:
Same comment as on #2431, can we set the query string in this `if/else` and have a single `m.dbMap.Select` call below.